<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLineEdit" name="textNome">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid red;
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>Nome</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLineEdit" name="textCognome">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid red;
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>Cognome</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLineEdit" name="textEmail">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid red;
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLineEdit" name="textPassword">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid red;
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLineEdit" name="textCPassword">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid red;
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>Conferma Password</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLineEdit" name="textTelefono">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid red;
padding-bottom:7px;</string>
     </property>
     <property name="placeholderText">
      <string>Telefono</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="error">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="reg_cliente_button">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Registrati</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
